cmd/tether/attach.go

```diff
@@ -51,34 +51,22 @@ type attachServerSSH struct {
 	sshConfig *ssh.ServerConfig
 
 	enabled bool
+
+	running bool
```

Comment:
I cannot see why we have running as well as enabled - as best I can tell they are both used to make start & stop idempotent and are effectively identical.